core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -383,16 +384,17 @@ class SpendReportingService(
                        |    spend_category,
                        |    currency""".stripMargin.trim
 
-    val bpSubQuery = billingProjects
-      .map { bp =>
-        val tableName = bp._1.spendExportTable.getOrElse(spendReportingServiceConfig.defaultTableName)
-        val timePartitionColumn: String = getTimePartitionColumn(tableName)
-        baseQuery
-          .replace("_PARTITIONTIME", timePartitionColumn)
-          .replace("_BILLING_ACCOUNT_TABLE", tableName)
-          .replace("_PROJECT_ID_LIST", "(" + bp._2.map(tuple => s""""${tuple._1.value}"""").mkString(", ") + ")")
-      }
-      .mkString("\nUNION ALL\n")
+//    val bpSubQuery = billingProjects
+//      .map { bp =>
+    val tableName = billingProject.spendExportTable.getOrElse(spendReportingServiceConfig.defaultTableName)
+    val timePartitionColumn: String = getTimePartitionColumn(tableName)
+    val bpSubQuery =
+      baseQuery
+        .replace("_PARTITIONTIME", timePartitionColumn)
+        .replace("_BILLING_ACCOUNT_TABLE", tableName)
+        .replace("_PROJECT_ID_LIST", "(" + workspaces.map(tuple => s""""${tuple._1.value}"""").mkString(", ") + ")")
```

Comment:
```suggestion
        .replace("_PROJECT_ID_LIST", "(" + workspaces.map { case (project, _) => s""""${project.value}"""" }.mkString(", ") + ")")
```
the `_1` syntax is tough, using the curlies and case make it look nicer